src/main/java/de/retest/web/WebDataFilter.java

```diff
@@ -0,0 +1,28 @@
+package de.retest.web;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+// TODO This should be part of recheck and somehow use RecheckAdapter#convert to filter the result.
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=94&issues=AWbuNaawTJjFgJLTqtx6&open=AWbuNaawTJjFgJLTqtx6)

[ISSUE_KEY]: AWbuNaawTJjFgJLTqtx6